core/unix/module_elf.c

```diff
@@ -1616,3 +1618,28 @@ module_relocate_rela(app_pc modbase, os_privmod_data_t *pd, ELF_RELA_TYPE *start
     for (rela = start; rela < end; rela++)
         module_relocate_symbol((ELF_REL_TYPE *)rela, pd, true);
 }
+
+void
+/* This routine is duplicated in privload_relocate_relr for relocating
+ * dynamorio symbols in a bootstrap stage. Any update here should be also
+ * updated in privload_relocate_relr.
+ */
+module_relocate_relr(app_pc modbase, os_privmod_data_t *pd, const ELF_WORD *relr,
+                     size_t size)
+{
+    ELF_ADDR *r_addr = NULL;
+
+    LOG(GLOBAL, LOG_LOADER, 4, "%s walking relr %p-%p\n", relr, (char *)relr + size);
```

Comment:
The %s has no arg: I'm surprised the compiler doesn't warn about mismatched types.  Probably this is meant to pas `__FUNCTION__`.

Reply:
To expedite things I went and edited this directly.  I see two other logs above are also missing the arg; fixed.